src/ds/ds-widget.js

```diff
@@ -120,13 +111,35 @@ jQuery(function ($) {
                         } else if (opts.maxResults > 0 && results.length > opts.maxResults) {
                             render(obj.options.too_many_results(this, results.length))
                         } else {
-                            for (let i in results) {
-                                let data = results[i]
-                                counter += 1;
-                                data.counter = counter;
-                                data.saved = false;
-                                render(obj.options.render_search_result(data))
+                            const getResults = function() {
+                                const numberDisplayed = $(obj.options.search_result_selector + " > *").length
+
+                                if (numberDisplayed === 0) {
+                                    return results.slice(0, 24)
+                                } else {
+                                    return results.slice(numberDisplayed, numberDisplayed + 25)
+                                }
+                            }
+
+                            const displayResults = function() {
```

Comment:
displayResults() appends the results in search_result_selector. This functionality seems to be handled in bootstrap-list-filter.src.js.